### ISSUE TITLE
BAF Wallet Logo

### ISSUE THREAD

**Issue by Sladuca** [bafnetwork/baf-wallet-v2#60]
Docusaurus puts its logo all over the place by default in the docs, and we'll probably want a logo eventually for branding reasons.

Idk if we want to just use the BAF Logo or have something else? I'm all ears for suggestions.

**Comment by Sladuca** [comment]
@cameron-dennis

**Comment by JMoss-spec** [comment]
I actually love the BAF logo 100%🥰

**Comment by Sladuca** [comment]
Yeah the BAF logo is great, but not sure if we want to just verbatim use the BAF logo for BAF Wallet, ya know?
